script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -205,15 +205,15 @@ But the teachers rely on you because you do that. I hope you don't let them get
 <Page63>
 "It's okay, I do it because I like it.# Besides, the teachers are thinking about the school. I don't like the way you're talking about them, Inui-kun."
 <Page64>
-Ciel-senpai rebukes Arihiko's lazy opinions.
+Ciel-senpai completely rebukes Arihiko's lazy opinions.//だらけきった有彦の意見を、正面からばっさりたしなめるシエル先輩。
 From the looks of it, he seems to be close to her in his own way.
 <Page65>
 "Senpai, do you do volunteer work every day by any chance?"
 "What, you don't know? Ciel-senpai here is such a handy person that she's called the true student council president."
 <Page66>
-"I know. She's the perfect third year who can solve anything, unlike the student council which is just a title at best."
+"I know. She's the perfect third year who can solve anything, unlike the so-called student council."//知ってる。肩書きだけの生徒会と違って、なんでも解決してくれる完璧な三年生だろ」If anyone has better ideas on how to phrase this
```

Comment:
Titular student council? Can't immediately come up with something better that doesn't involve pulling extra meaning out of nowhere, e.g. 
> Unlike that useless lot that are student council members in name only, she's a perfect third year who can solve anything.
> Unlike the members of the student council, who only [help students?] on paper, she's a perfect third year who can solve anything.

Reply:
I think so-called is fine

Reply:
Yeah that was my problem too.
>Unlike the members of the student council, who only help students on paper, she's a perfect third year who can solve anything.

That's a nice sentence, I like it, but it does add a little.
Well, if everyone is fine with "so-called", I'll keep it.